Packs/Slack/Playbooks/playbook-Slack_BlockBuilderExample_README.md

```diff
@@ -0,0 +1,38 @@
+This is an example of how to use the SlackBlockBuilder within a playbook.
+
+## Dependencies
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+This playbook does not use any sub-playbooks.
+
+### Integrations
+
+This playbook does not use any integrations.
+
```

Comment:
do we really need all of this? is it because of a validation?

Reply:
:hide-the-pain: yeah it's all part of build validations these days.